app/domain/authentication/authn_jwt/authenticator.rb

```diff
@@ -7,7 +7,11 @@ module AuthnJwt
     Authenticate = CommandClass.new(
       dependencies: {
         token_factory: TokenFactory.new,
-        logger: Rails.logger
+        logger: Rails.logger,
+        audit_log: ::Audit.logger,
+        validate_origin: ::Authentication::ValidateOrigin.new,
+        role_cls: ::Role,
+        webservice_class: ::Authentication::Webservice,
```

Comment:
choose between "cls" and "class" and consist with it

Reply:
Thanks. Choosed class